runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -37,6 +37,7 @@
 #define MAX_BACKEND_NAME_LENGTH 32
 #define MAX_OP_NAME_LENGTH 64
 #define MAX_PATH_LENGTH 1024
+#define MAX_TENSOR_NAME_LENGTH 64
```

Comment:
@wateret I see some tensors has name of 80+. However, it is used only for input and output tensor of model. It may be okay.

Reply:
Then I'll set it to 256!

Reply:
```suggestion
#define MAX_TENSOR_NAME_LENGTH 256
```

Reply:
For input, output tensor, it will be OK (for internal tensor, there is 225 char-lengh name)